x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/search_panel/search_panel.scss

```diff
@@ -13,8 +13,4 @@
   .dvSearchPanel__controls {
     padding: 0;
   }
-  // prevent margin -16 which scrunches the filter bar
-  .globalFilterGroup__wrapper-isVisible {
```

Comment:
@stratoula Looks like this is stemming from the [same problem I encountered in Discover ](https://github.com/elastic/kibana/pull/128401#discussion_r858871341)where `displayStyle` is not getting passed down when the search bar is implemented via `services.unifiedSearch.ui.SearchBar`.

Reply:
Fixed https://github.com/elastic/kibana/pull/128401/commits/1bb10650587963e5771cd617dd915ba2f30d179c

Reply:
@qn895 It's not great because there's a lot of controls on that single line, but it's aligned now
<img width="1262" alt="Screen Shot 2022-04-26 at 17 46 45 PM" src="https://user-images.githubusercontent.com/549577/165398025-6f3e0574-2bd8-4daa-ab48-544a0b925d0d.png">